dblookupext/historyRepository_test.go

```diff
@@ -8,8 +8,11 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/core"
 	"github.com/ElrondNetwork/elrond-go-core/data"
 	"github.com/ElrondNetwork/elrond-go-core/data/block"
+	mock3 "github.com/ElrondNetwork/elrond-go-core/data/mock"
```

Comment:
please replace `mock2` and `mock3` with something else

Reply:
Renamed them